collect_app/src/main/java/org/odk/collect/android/dao/CursorLoaderFactory.java

```diff
@@ -31,27 +48,13 @@ public CursorLoader createSentInstancesCursorLoader(CharSequence charSequence, S
         return cursorLoader;
     }
 
-    public CursorLoader createSentInstancesCursorLoader(String sortOrder) {
```

Comment:
I believe the only thing that changed in this section of the diff is method order but it's not displayed very well. Is that right?